### DISCUSSION TITLE
Static deflection during operation

### DISCUSSION THREAD

**Discussion by LeifParr** [petrobras/ross#870] [Q&A] [answered]
We are in the process of trouble-shooting a vibration issue in a horizontal multistage pump. One line of inquiry is that the bearing(s) are unloaded due to static lift of seals. Our software RAPPID does not have any capabilities for generating a static deflection during operation. I do not immediately see how it may be done using ROSS either. Has anyone of you done a similar investigation using ROSS? 
The idea is to see if we can recreate the shaft centerline plot (see below).
![image](https://user-images.githubusercontent.com/74429840/148898091-94038b96-e9f5-4e5d-af4d-d5fc97b65024.png)

**Comment by raphaeltimbo** [comment, marked as answer]
@LeifParr, with ROSS it is possible to generate a deflected shape from a forced response analysis (e.g. unbalance response). 
An example in the tutorial: https://ross.readthedocs.io/en/stable/tutorials/tutorial_part_2.html#plotting-results-deflected-shape-a-id-section1-5-3-a
But to do this, you would need to estimate the forces acting in the rotor.

**Reply by LeifParr** [reply]
Thank you for your thoughts on this issue. I looked up the documentation, as far as I can see, the unbalance response deals with response to a rotating force. I was thinking about an extension of the static type of analysis - for slender rotors at high speeds maybe the gyroscopic moment is significant, maybe possibilities for defined physical offset of bearings etc. I do think MADYN has these capabilities. For now, I have used a simple static beam calculation to investigate the effects of bearing offset, disregarding the dynamic factors.
